front/src/Connexion/LocalUserStore.ts

```diff
@@ -160,19 +164,24 @@ class LocalUserStore {
 
     verifyState(value: string): boolean {
         const oldValue = localStorage.getItem(state);
-        localStorage.removeItem(state);
         return oldValue === value;
     }
+    getState(): string | null {
```

Comment:
state and nonce should never be reused. there are supposed to be newly created at each login attempt.

Reply:
Yes, I know it will be change